### DISCUSSION TITLE
Changing sprites

### DISCUSSION THREAD

**Discussion by NIOBEG-ALT** [KadeArchive/Kade-Engine#1409] [Q&A]
Uhh does Anyone know how to change sprites in song? Like in vs nonsense mod

**Comment by raf13lol** [comment]
modcharts
example:

-- this gets called starts when the level loads.
function start(song) -- arguments, the song name

end

-- this gets called every frame
function update(elapsed) -- arguments, how long it took to complete a frame

end

-- this gets called every beat
function beatHit(beat) -- arguments, the current beat of the song

end

-- this gets called every step
function stepHit(step) -- arguments, the current step of the song (4 steps are in a beat)

if curStep == 69 then
changeBoyfriendCharacter('pico')

if curStep == 69 then
changeDadCharacter('pico')

end

this changes the boyfriend and the enemy into pico after 69 steps or 17.25 beats

**Reply by EcoVeni** [reply]
i am no longer on alt but thanks!
